VMBackup/main/handle.py

```diff
@@ -317,6 +322,10 @@ def daemon():
         para_parser = ParameterParser(protected_settings, public_settings, backup_logger)
         hutil.update_settings_file()
 
+        if(para_parser.taskId is not None and para_parser.taskId != ""):
+            eventlogger.update_properties(para_parser.taskId)
+        hutil.set_event_logger(eventlogger)
```

Comment:
why is L:327 required when we are already setting the event logger in L82?

Reply:
discussed offline